packages/clerk-js/src/core/resources/Client.ts

```diff
@@ -83,6 +85,7 @@ export class Client extends BaseResource implements ClientResource {
       this.signUp = new SignUp(data.sign_up);
       this.signIn = new SignIn(data.sign_in);
       this.lastActiveSessionId = data.last_active_session_id;
+      this.cookieExpiresAt = data.cookie_expires_at ? unixEpochToDate(data.cookie_expires_at) : null;
```

Comment:
How valid is this, for environments where cookies do not exist ?
For example `clerk-js` loaded via expo.

Reply:
It will be a no-op in non-browser environments and users will not be navigated anywhere 👌